.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,38 @@
+
+<!--
+Thank you for your interest in contributing to XCSoar! Please fill out the
+following questions to make it easier for us to review your changes.
+
+You do not need to check all the boxes below all at once, feel free to take
+your time and add more commits. If you're done and ready for review, please
+check the last box.
+-->
+
+
+What is the purpose of this pull request? What does it change?
+--------------------------------------------------------------
+
+<!--
+Please describe the changes here, as detailed as needed.
+-->
+
+
+Has the changes been discussed in an issue or the forum?
+--------------------------------------------------------
+
+<!--
+Please link issues and relevant forum posts here.
+
+If this PR resolves an issue on GitHub, write "Closes #1234" so that the issue
+is closed automatically when this PR is merged.
+-->
+
+
+Checklist
+---------
+
+- [ ] I have enabled [maintainer edits for this PR](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork)
+- [ ] I have added tests for all changes in this PR
+- [ ] I have added documentation for the changes (in the manual)
+- [ ] I've formatted commit messages in the same style as [other commits in the repo](https://github.com/XCSoar/XCSoar/commits/master)
+- [ ] I'm done, this Pull Request is ready for review
```

Comment:
Of course it is. Why expect people to check this? This is redundant - there's already the button to submit the PR which means exactly this.

Reply:
This checkbox is there to indicate that (if unchecked) the PR is a draft that's being worked on (which can happen if the author want to let others look at and comment on things before he considers it ready for review).

I don't think there's any other checkbox or similar in the PRs to indicate whether it's a work in progress, so either we don't need to know that, or we rely on people adding e.g. `[WIP]` in the title, or we keep this checkbox. I'm fine with either.